### PR TITLE
Battleintro: Skip with mouse button

### PR DESCRIPTION
**Pull request by Laserlicht**

### CODE REVIEW

File: client/battle/BattleWindow.cpp [hunk 1]
```diff
@@ -204,6 +204,16 @@ void BattleWindow::keyPressed(EShortcut key)
...
 }
 
+void BattleWindow::clickPressed(const Point & cursorPosition)
+{
+	if (owner.openingPlaying())
+	{
+		owner.openingEnd();
+		return;
+	}
+	InterfaceObjectConfigurable::clickPressed(cursorPosition);
```
Comment (dydzio0614) [inline client/battle/BattleWindow.cpp:214]: shouldn't order be reversed (move before "if") so single click won't trigger battlefield action after opening ends?

Comment (Laserlicht) [reply client/battle/BattleWindow.cpp:214]: It is exact the same logic as for "keyPressed".

I don't understand what you mean. After opening ends the function ignores the action with "return".

Comment (dydzio0614) [reply client/battle/BattleWindow.cpp:214]: at yeah, nevermind

**Review by IvanSavenko — APPROVED** [review]
Will also work for touchscreen / mobiles, where you can't just "press any key"

**Comment by Laserlicht** [comment]
> Will also work for touchscreen / mobiles, where you can't just "press any key"

Like "no keyboard present - press any key to continue..." 😀

**Pull request merged**